.github/pull_request_template.md

```diff
@@ -1,41 +1,9 @@
-<!--  Thanks for sending a pull request!  Before submitting:
-
-1. Read our CONTRIBUTING.md guide
-2. Name your PR as `<Feature Area>: Describe your change`.
-  a. Do not end the title with punctuation. It will be added in the changelog.
-  b. Start with an imperative verb. Example: Fix the latency between System A and System B.
-  c. Use sentence case, not title case.
-  d. Use a complete phrase or sentence. The PR title will appear in a changelog, so help other people understand what your change will be.
-3. Rebase your PR if it gets out of sync with main
--->
-
 **What this PR does / why we need it**:
```

Comment:
I think we can add these sections back in, as they'd be relevant in the comments as well potentially - and they give a nice structure to the PR description.

The rest of the cruft can go to the CONTRIBUTING.md which you've done

```suggestion
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:
```